index.html

```diff
@@ -25,7 +25,8 @@ <h1>Skiafy</h1>
     <p>Options:</p>
     <p>
       <label><input id="flip-x" type="checkbox"> Flip on <var>x</var> axis</label><br/>
-      <label><input id="flip-y" type="checkbox"> Flip on <var>y</var> axis</label>
+      <label><input id="flip-y" type="checkbox"> Flip on <var>y</var> axis</label><br/>
+      <label><input id="preserve-fill" type="checkbox"> Try to preserve fill color<span class="limitation"> (works only with fill attributes & expects colors in form #FFF or #FFFFFF).</span></label>
```

Comment:
We should note something along the lines of "This is non-default because usually it's preferable to specify an icon's color at runtime rather than in the .icon definition."